data/client/king-phisher.desktop

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=King Phisher
 GenericName=King Phisher
-Exec=python /opt/king-phisher/KingPhisher
+Exec=/usr/bin/python3 /opt/king-phisher/KingPhisher
```

Comment:
Jul 26 16:20:18 ubuntu16-test systemd[1]: [/etc/systemd/system/king-phisher.service:12] Executable path is not absolute, ignoring: python3 /opt/king-phisher/KingPhisherServer /opt/king-phisher/server_config.ym
Jul 26 16:20:18 ubuntu16-test systemd[1]: king-phisher.service: Service has no ExecStart= setting, which is only allowed for Type=oneshot services. Refusing.
Jul 26 16:20:36 ubuntu16-test systemd[1]: Stopping King Phisher Server...
Jul 26 16:20:36 ubuntu16-test systemd[1]: Stopped King Phisher Server.
Jul 26 16:20:46 ubuntu16-test systemd[1]: [/etc/systemd/system/king-phisher.service:12] Executable path is not absolute, ignoring: python3 /opt/king-phisher/KingPhisherServer /opt/king-phisher/server_config.yml